src/components/DarkModeToggle.js

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from 'react';
+import '../App.css'; // Make sure this path is correct
+
+const DarkModeToggle = () => {
+  const [isDarkMode, setIsDarkMode] = useState(false);
+
+  useEffect(() => {
+    if (isDarkMode) {
+      document.body.classList.add('dark-theme');
+    } else {
+      document.body.classList.remove('dark-theme');
+    }
+  }, [isDarkMode]);
+
+  const handleToggle = () => {
+    setIsDarkMode(!isDarkMode);
+  };
+
+  return (
+    <button 
+      onClick={handleToggle} 
+      aria-label={isDarkMode ? "Switch to light mode" : "Switch to dark mode"}
+      className="dark-mode-toggle-button"
+    >
+      <img
+        src={isDarkMode ? "sun.png" : "moon.png"}
```

Comment:
Unfortunately, these icons don't work. To make them work, you should provide a path to them (or import them on the top of the file) 😊 

![Screenshot 2024-07-11 at 9 59 31](https://github.com/paularynty/todolist/assets/49838422/14061656-e56f-4220-83da-e656ae10fbe6)